### ISSUE TITLE
[RVR Quest] Praag scouting objectives quest "In Shadows" bugged.

### ISSUE THREAD

**Issue by TVBrowntown** [WarEmu/WarBugs#20352]
Scouting Kurlov's Armory in Praag for the repeatable quest **In Shadows** fails to trigger completion.

**Steps to reproduce the problem:**
1. Take quest In Shadows as Order
2. Go to KA in Praag

**Testing Screenshots/Videos/Evidence (always needed):**

![WAR_2022 08 19-13 15](https://user-images.githubusercontent.com/32625383/185538600-2139bcd9-0e4e-40d5-8f66-c033cd9cac2c.png)

**Comment by TVBrowntown** [comment]
It's possible this quest only functions when the zone is active? Other areas were not able to be scouted either. Not entirely sure when MS was ticked off in my quest.

**Comment by xpander69** [comment]
Yeah, zones have to be contested afaik

**Comment by TVBrowntown** [comment]
Very reasonable. Quest texts for these could be updated to include the information that only apply to active zones. Additionally, the quests to scout the keeps are able to be done even when the zone is currently locked.

**Comment by xpander69** [comment]
True, but probably super low priority, unless its easy to fix those. Thanks for reporting.
